hooks/boot/01-helpers.rb

```diff
@@ -39,17 +39,17 @@ def execute(commands, do_say = true)
       !results.include? false
     end
 
-    def execute_command(command, do_say)
+    def execute_command(command, do_say, do_log)
```

Comment:
How does Ruby handle a function without a default? I.e. will line 37 break or assume false?

Reply:
Good catch, I updated the file